storage/mdadm/pkg.yaml

```diff
@@ -0,0 +1,39 @@
+name: mdadm
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+  - image: "{{ .PKGS_PREFIX }}/eudev:{{ .BUILD_ARG_PKGS }}"
+  - image: "{{ .PKGS_PREFIX }}/util-linux:{{ .BUILD_ARG_PKGS }}"
```

Comment:
`util-linux` is already part of Talos, so we can drop it

Reply:
same with `eudev`

Reply:
They required by gcc, to build mdadm...

Reply:
nevermind, i thought this was the final stage